axelor-account/src/main/resources/views/Move.xml

```diff
@@ -456,14 +456,14 @@
       </panel>
     </panel>
 
-    <panel>
+    <panel hideIf="!journal">
```

Comment:
I think a panel containing a panel related is useless. You can remove it and add the `hideIf` condition to the panel related